docker/gcp_pudl_etl.sh

```diff
@@ -149,8 +117,11 @@ function zenodo_data_release() {
 
     if [[ "$1" == "production" ]]; then
         ~/pudl/devtools/zenodo/zenodo_data_release.py --no-publish --env "$1" --source-dir "$PUDL_OUTPUT"
-    else
+    elif [[ "$1" == "sandbox" ]]; then
```

Comment:
Yes that's right -- there's still some manual metadata / description update stuff that needs to happen and we want a set of human eyeballs on it to look for anything funny since we can't easily revoke archives.